server/core/runtime/post_workflow_hook_runner.go

```diff
@@ -23,7 +23,7 @@ type DefaultPostWorkflowHookRunner struct {
 func (wh DefaultPostWorkflowHookRunner) Run(ctx models.WorkflowHookCommandContext, command string, path string) (string, string, error) {
 	outputFilePath := filepath.Join(path, "OUTPUT_STATUS_FILE")
 
-	cmd := exec.Command("sh", "-c", command) // #nosec
+	cmd := exec.Command("bash", "-c", command) // #nosec
```

Comment:
This is a large change. If you want to include this, we'd need it in a separate pr and with a switch to override it. That switch would have to default to sh to not break existing pipelines

Reply:
This has already been reverted.